libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/RouteConstants.java

```diff
@@ -1,6 +1,7 @@
 package com.mapbox.navigation.ui.route;
 
 class RouteConstants {
+  static final String DEFAULT_ROUTE_LINE_LAYER_BELOW_ID = "road-label";
```

Comment:
The Nav SDK cannot default to this layer id because a custom Map style might not have it. This would mean, that the route is most-likely end up being drawn on top of everything, including the puck.

Reply:
What layer should be used?

Reply:
The ideal fallback flow should be:
provided layer id -> `LocationComponentConstants.SHADOW_LAYER` (**if the layer exists**) -> `LocationComponentConstants.FOREGROUND_LAYER` (**if the layer exists**) -> `null`